.secrets.baseline

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
How was this file generated?

Reply:
@mhassaninmsft  The file was generated by the detect-secrets module. I executed 'detect-secrets scan > .secrets.baseline'. An initial baseline file is required when running the tool. Once this PR is approved my plan is to rebase and generate/commit an updated baseline file prior to completing the PR.